\begin{tabularx}{\textwidth}{ L }
  \textbf{PUBLICATIONS} \\
\end{tabularx}

\begin{tabularx}{\textwidth}{ X }
    Grossman R., Heath A., \textbf{Murphy M.}, Patterson M., Wells W. \\
    \footnotesize\textit{A Case for Data Commons: Toward Data Science as a Service} \\
    \footnotesize{\textit{Computing in Science \& Engineering}, vol. 18, no. 5, pp. 10-20, Sept.-Oct. 2016} \\

    \vspace{1pt}

    Frankhouser D., \textbf{Murphy M.}, Blachly J., Park J., Zoller M., \textit{et al.} \\
    \footnotesize\textit{PrEMeR-CG: inferring nucleotide level DNA methylation values from MethylCap-seq data} \\
    \footnotesize{\textit{Bioinformatics}, vol. 30, iss. 24, pp. 3567--3574, Dec. 2014} \\

    \vspace{1pt}

    Yan P., Frankhouser D., \textbf{Murphy M.}, Tam H., Rodriguez B., \textit{et al.} \\
    \footnotesize\textit{Genome-wide methylation profiling in decitabine-treated patients with acute myeloid leukemia} \\
    \footnotesize{\textit{Blood}, vol. 120, iss. 12, pp. 2466--2474, Sept. 2012} \\

    \vspace{1pt}

    Trimarchi M., \textbf{Murphy M.}, Frankhouser D., Rodriguez B., \textit{et al.} \\
    \footnotesize\textit{Enrichment-based DNA methylation analysis using next-generation sequencing:} \\
    \footnotesize\textit{Sample exclusion, estimating changes in global methylation, and the contribution of replicate lanes} \\
    \footnotesize{\textit{BMC Genomic}, vol. 13, suppl. 8, s. 6, Dec. 2012} \\
\end{tabularx}

